include/iocore/eventsystem/UnixSocket.h

```diff
@@ -92,34 +95,33 @@ class UnixSocket
   int set_sndbuf_size(int bsz);
   int set_rcvbuf_size(int bsz);
 
+  int enable_option(int level, int optname);
+
   int close();
   int shutdown(int how);
 
   static bool client_fastopen_supported();
-
-private:
-  int sock_fd{NO_SOCK};
 };
 
-inline UnixSocket::UnixSocket(int fd) : sock_fd{fd} {}
+inline UnixSocket::UnixSocket(int fd) : fd{fd} {}
 
 inline UnixSocket::UnixSocket(int domain, int type, int protocol)
 {
-  this->sock_fd = socket(domain, type, protocol);
+  this->fd = socket(domain, type, protocol);
```

Comment:
I think we should remove all the instances of `this->` unless a local variable is shadowing a class member.

Reply:
We've decided to leave this because leave the file is new and doesn't have a different existing practice.